back/app/controllers/web_api/v1/files_controller.rb

```diff
@@ -120,9 +120,20 @@ def set_container
     @container = secure_constantize(:container_class).find container_id
   end
 
-  def transform_errors_details!(error_details)
+  def generate_error_details(errors)
+    # @file.errors.details always returns { error: ActiveModel::Error#type }
```

Comment:
Before Carrierwave upgrade, the errors looked like this
```
> @file.errors
=> #<ActiveModel::Errors [
  #<ActiveModel::Error attribute=file, type=max_size_error, options={}>, 
  #<ActiveModel::Error attribute=file, type=blank, options={:unless=>#<Proc:0x00005638788ab5e0 /cl2_back/app/models/project_file.rb:30>}>
]>
```

Reply:
Here's how these Carrierwave messages are generated https://github.com/carrierwaveuploader/carrierwave/blob/f5b09b844d99245a3b4d0ba01efd4972be4ee5be/lib/carrierwave/uploader/file_size.rb#L36

Here's how `details` are generated https://github.com/rails/rails/blob/main/activemodel/lib/active_model/error.rb#L150